Assets/Mirror/Runtime/Transport/FallbackTransport.cs

```diff
@@ -58,9 +58,27 @@ void InitClient()
             }
         }
 
-        public override void ClientConnect(string address)
+        public override void ClientConnect(Uri uri)
         {
-            available.ClientConnect(address);
+            foreach (Transport transport in transports)
```

Comment:
why does this go through all transports again to find the first available one? that's what .available is for already

Reply:
Because the transport that is selected depends on the scheme in the uri.   

Maybe the first available does not support the uri provided,  we need to try the other ones to see if one of them succeeds.